packages/react-magma-dom/src/components/ToggleButton/ToggleButtonGroup.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+
+import { ButtonColor, ButtonSize } from '../Button';
+import { ButtonGroup } from '../ButtonGroup';
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import {
+  ToggleButtonStyles,
+  ToggleButton,
+  ToggleButtonProps,
+} from './ToggleButton';
+
+/**
+ * @children required
+ */
+export interface ToggleButtonGroupProps
+  extends Omit<
+    React.HTMLAttributes<HTMLButtonElement>,
+    'isDefault' | 'isChecked'
+  > {
+  isChecked?: boolean;
+  isDefault?: boolean;
+  isInverse?: boolean;
+  /**
+   * Removes margins between buttons and applies a uniform border around the group.
+   */
+  noSpace?: boolean;
+  /**
+   * Enables a radio configuration throughout the group retaining an active selection at all times.
+   */
+  requiredSelect?: boolean;
+  /**
+   * Sets the Toggle Button group to have only one active selection.
+   */
+  singleSelect?: boolean;
+  /**
+   * Changes the button sizes throughout the group between 'small', 'medium', and 'large'.
+   */
+  size?: ButtonSize;
+  /**
+   * @internal
+   */
+  testId?: string;
+  /**
+   * @internal
+   */
+  theme?: ThemeInterface;
+}
+
+export interface ToggleButtonGroupContextInterface {
+  isDefault?: boolean;
+  isInverse?: boolean;
+  requiredSelect?: boolean;
+  isChecked?: boolean;
+  singleSelect?: boolean;
+  size?: ButtonSize;
+}
+
+export const ToggleButtonGroupContext =
+  React.createContext<ToggleButtonGroupContextInterface>({});
+
+const SingleSelectWrapper = styled.div<{
+  isDefault?: boolean;
+  isInverse?: boolean;
+  requiredSelect?: boolean;
+  isChecked?: boolean;
+  singleSelect?: boolean;
+  size?: ButtonSize;
+  value?: any;
```

Comment:
ToggleButtonStyles only uses some of the props, so we can remove the extra ones from here.

Reply:
This was causing issues on docs and has been removed here.